app/templates/single_resource/edit.html

```diff
@@ -1,10 +1,12 @@
 {% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
 
 {% block content %}
     <div class="ui text container">
         <h2 class="ui dividing header">Edit Resource</h2>
         {% include 'single_resource/resource_form.html' %}
-        <a href=""
-           class="ui negative deletion button">Delete</a>
+        <br>
+        <a href="{{ url_for('single_resource.delete', resource_id=resource_id) }}"
```

Comment:
I wonder if this should be a POST request (via ajax) instead of a POST request? Could be a tiny bit safer than using GET requests.